neuraxle/base.py

```diff
@@ -414,14 +435,17 @@ def mkdir(self):
         if not os.path.exists(path):
             os.makedirs(path)
 
-    def get_path(self):
+    def get_path(self, with_root=True):
         """
         Creates the directory path for the current execution context.
 
         :return: current context path
         :rtype: str
         """
-        parents_with_path = [self.root] + [p.name for p in self.parents]
+        parents_with_path = [self.root] if with_root else []
+        parents_with_path += [p.name for p in self.parents]
+        if len(parents_with_path) == 0:
+            return '/'
```

Comment:
This is to be fixed. 

`rm -rf /` ? This line of code here seems dangerous. It should probably use a tmp folder: https://docs.python.org/2/library/tempfile.html

Or maybe you meant `./` instead of `/` but anyway your code won't work on windows in both cases here, as you should use `os.sep` instead. Not sure what you meant to do here but "/" seems dangerous.